Sub section to intro course

Lymah123 opened the pull request:
## This PR makes changes to the following:
- #59 

<!-- 
Please do not leave this blank 
This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 
-->

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
<!-- 
Please use this format link issue numbers: Fixes #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

## Mobile & Desktop Screenshots/Recordings

<!-- Visual changes require screenshots -->


## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?



<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
  
  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

BekahHW reviewed (changes requested):
This is a great start!

Make sure you follow the style conventions of the other chapters. 

Let's start with those changes.

1. All the files are markdown files. Let's rename this `types-of-contributions.md`
2. Take a look at how markdown is used. Use [this style guide](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md) to help you make decisions about how to use section headings and other markdown styles.

Lymah123 commented:
Alright. Thank you!

Lymah123 commented:
I have changed the file to md.

BekahHW reviewed (changes requested):
Hey @Lymah123. There should be one file for the types of contributions. 

So you'll need to remove TYPES OF CONTRIBUTIONS.pdf, and Types of Contributions.

You'll need to change the name of the md file as well.  It needs to include the -. Here's what you can do:

`07-types-of-contributions.md`. 

Because there's already a 07 section in the course, you'll need to update the subsequent chapters to 08, 09, 10.

Does that make sense?

CBID2 commented on Types of Contributions:
```diff
@@ -0,0 +1,264 @@
+
+Non-coding contributions are valuable contributions made to open source projects that do not involve writing or modifying code. These contributions are essential for the overall health and success of open source communities and projects. They help improve documentation, user experience, project management, and community engagement.
+
+Here are some common types of non-coding contributions
```
```suggestion
Here are some common types of non-coding contributions:
```

CBID2 commented on Types of Contributions:
```diff
@@ -0,0 +1,264 @@
+
+Here are some common types of non-coding contributions
+
+Documentation
+Graphic design
+Blog writing
+
```
[proofreading] Consider presenting the list in bullet point form and hyperlinking each item. It'll make it easier for users to navigate! 
```suggestion

- [Documentation](#Documentation)
- [Graphic Design](#Graphic Design)
- [Blog Writing](#Blog Writing)

```

CBID2 commented on Types of Contributions:
```diff
@@ -0,0 +1,264 @@
+Blog writing
+
+
+DOCUMENTATION
+
+Documentation is a crucial type of open source contribution that involves creating, improving, or maintaining the written resources that accompany open source software projects. Good documentation can make a project more accessible and user-friendly, and it can help attract new contributors.
+
+Contributing to documentation in open source projects is a valuable way to make a meaningful impact, even if you're not a developer. Clear and well-organized documentation enhances the overall quality of the project and fosters a more welcoming and inclusive open source community.
```
[Proofreading] While the second paragraph is helpful in emphasizing the benefits of docs-related contributions, consider omitting it for clarity. I also recommend beginning the paragraph with a statement that draws people's interest in this specific type of OS contribution. 
```suggestion
## Documentation
Are you a lover of words? Do grammar errors irritate you like seeing pineapple on pizza? Then, documentation is for you! 
This involves creating, improving, or maintaining the written resources that accompany open source software projects. Contrary to popular belief, documentation is crucial to open source projects. It can make a project more accessible and user-friendly, and it can help attract new contributors.

CBID2 commented on Types of Contributions:
```diff
@@ -0,0 +1,264 @@
+Contributing to documentation in open source projects is a valuable way to make a meaningful impact, even if you're not a developer. Clear and well-organized documentation enhances the overall quality of the project and fosters a more welcoming and inclusive open source community.
+
+
+Importance of good documentation
+
+Ease of Use: Well-documented open source projects are more accessible to users and developers. Clear and comprehensive documentation helps users understand how to install, configure, and use the software effectively.
+
+Onboarding: Documentation is essential for onboarding new contributors to a project. It provides them with the necessary information to get started, including project goals, coding standards, and development workflows.
+
+
+Troubleshooting: When users or developers encounter issues or bugs, documentation often contains troubleshooting guides and FAQs that can help them resolve problems without needing to contact the project maintainers.
+
+Knowledge Sharing: Documentation is a means of sharing knowledge and best practices within the open source community. It allows experienced developers to pass on their expertise to others and ensures that knowledge is not lost as contributors come and go.
+
+Community Growth: High-quality documentation can attract more users and contributors to an open source project. It builds trust and confidence in the project, which can lead to a larger and more active community.
+
+Maintainability: Documentation can also encompass guides on how to contribute to the project, including coding guidelines and codebase architecture. This helps maintain consistency and makes it easier for developers to contribute effectively.
+
+
```
```suggestion
### How to Contribute to Documentation
If you're interested in contributing to documentation, here are a few tips:

Read the documentation: This will help you understand how the product is used.

Find a task that's a good fit for your skills and interests: If you're not sure where to start, ask the project's maintainers for suggestions.

Use the product: Gaining user insight will help you see which information is needed or not useful in the docs.  

Be patient and persistent: It takes time to write good documentation. Don't be afraid to ask for help if you get stuck.

```

CBID2 commented on Types of Contributions:
```diff
@@ -0,0 +1,264 @@
+
+
+
+If you're interested in contributing to documentation, here are a few tips:
+
+Start by reading the project's documentation and getting familiar with the project's code base. This will help you understand what the project needs and how you can contribute.
+
+Find a task that's a good fit for your skills and interests. If you're not sure where to start, ask the project's maintainers for suggestions.
+
+Be patient and persistent. It takes time to write good documentation. Don't be afraid to ask for help if you get stuck.
+
```
[proofreading] Consider replacing your current list with the information below:
```suggestion
Here are some specific examples of documentation contributions:

- Write a README file that explains how to install and use the project

Example: [Create a Bootstrap Example] 
(https://github.com/Mirrorful/mirrorful/pull/321)


- Create a tutorial that walks users through the project's features

Example: [An Answer to a thought-provoking question](https://insights.opensauced.pizza/feed/438)

- Do an explainer page of a product's feature 

Example: [Add Dev Cards to Getting Started](https://github.com/open-sauced/docs/issues/156)


- Translate the project's documentation into other languages

Example: [Translate the course to another language](https://github.com/open-sauced/intro/issues/16)


- Fixing typos and grammatical errors in the documentation

Example: [Making Directions Clearer](https://insights.opensauced.pizza/feed/456)

- Improving the formatting of the documentation

Example:[Incorrect Link Destination in Documentation](https://github.com/open-sauced/docs/pull/144)



- Adding missing information to the documentation

Example: [Add More Information to Use This Template in the README](https://github.com/open-sauced/100-days-of-oss-template/pull/8)

```

CBID2 reviewed (changes requested):
Hi @Lymah123! :) Your PR is off to a great start! :) I gave some very detailed tips on how to reorganize some information.

geoffreylgv reviewed (commented):
Well done @Lymah123 

I saw a commit message as `"Add files via upload"`
When you add files to GitHub via upload, you are not tracking the changes you're making to those files. This can make it difficult to revert to a previous version of the file or to see another contributor changes on the same file.

Lymah123 commented:
Thanks so much @BekahHW,  @CBID2 and @geoffreylgv 

I am committed to addressing all the points that have been raised and making the necessary corrections.

CBID2 reviewed (changes requested):
Hi @Lymah123. You’re making great progress! :) I provided more suggestions to your PR. Keep it up! :)

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+[
+Add More Information to Use This Template in the README]([url](https://github.com/open-sauced/100-days-of-oss-template/pull/8)https://github.com/open-sauced/100-days-of-oss-template/pull/8)
```
```suggestion
Add More Information to Use This Template in the README(https://github.com/open-sauced/100-days-of-oss-template/pull/8)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+- Adding missing information to the documentation
+
+*Example*
+[
```
```suggestion

```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Insights on social card generation]([url](https://github.com/open-sauced/opengraph/pull/51)https://github.com/open-sauced/opengraph/pull/51)
+
+- **Icon Design**: Icons are essential for user interfaces. Graphic designers can create custom icons for software applications or websites, improving the visual appeal and usability of the project.
+
+*Example*
+
+ [Redesign footer]([url](https://github.com/open-sauced/opengraph/pull/51)https://github.com/open-sauced/opengraph/pull/51)
```
```suggestion
[Insights on social card generation](https://github.com/open-sauced/opengraph/pull/51)

- **Icon Design**: Icons are essential for user interfaces. Graphic designers can create custom icons for software applications or websites, improving the visual appeal and usability of the project.

*Example*

 [Redesign footer](https://github.com/open-sauced/opengraph/pull/51)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Form page UI implemented
+]([url](https://github.com/zillBoy/pokemon-app/pull/6)https://github.com/zillBoy/pokemon-app/pull/6)
```
```suggestion
[Form page UI implemented
](https://github.com/zillBoy/pokemon-app/pull/6)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Add highlights to the user profile page]([url](https://github.com/open-sauced/app/pull/859)https://github.com/open-sauced/app/pull/859)
```
```suggestion
[Add highlights to the user profile page](https://github.com/open-sauced/app/pull/859)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+*Example*
+
+[Redesign footer
+]([url](https://github.com/cesko-digital/pomahejukrajine-web/pull/222)https://github.com/cesko-digital/pomahejukrajine-web/pull/222)
```
```suggestion
](https://github.com/cesko-digital/pomahejukrajine-web/pull/222)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+ [Improved the user experience of the highlights input form]([url](https://github.com/open-sauced/app/pull/1362)https://github.com/open-sauced/app/pull/1362)
```
```suggestion
 [Improved the user experience of the highlights input form](https://github.com/open-sauced/app/pull/1362)
```

Lymah123 commented:
Thank you @CBID2 !

I have made the necessary corrections you've mentioned.

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+*Example* 
+
+[Enhance feed layout responsiveness 
+]([url](https://github.com/open-sauced/app/pull/1553)https://github.com/open-sauced/app/pull/1553)
```
```suggestion
](https://github.com/open-sauced/app/pull/1553)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+### BLOG WRITING
+
+Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. These written contributions serve various purposes within the open-source community and can be valuable in several ways.
+
+Blog writing can be a great way to contribute to open-source projects, even if you don't have any coding skills. You can write about your experiences using the project or about the latest developments in the project. You can also write about the project's history, its community, or its impact on the world.
+
+Contributing blog posts to open-source projects or community websites is a valuable way to share knowledge, support the community, and raise awareness about important topics. It can also help open-source projects reach a wider audience and attract new contributors. Many open-source communities welcome blog contributions and have established platforms or publications where contributors can share t ...
```
```suggestion
Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. Here are some ways you can use blogging to contribute:

- Reflect on your experiences as a contributor
- Describe and review the project’s latest updates
- Mentioning the project when raising awareness about certain topics or in a guide on how to get started in open source.
 

```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+### *Here are some specific examples of how blog writing can be used to contribute to open-source projects:*
+
+- A blog post could be written about the latest features of an open-source project, such as a new version of the software or a new plugin.
```
```suggestion

```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+[Announcing the 0.2.0 release of Analog]([url](https://dev.to/analogjs/announcing-the-020-release-of-analog-aa1)https://dev.to/analogjs/announcing-the-020-release-of-analog-aa1)
+
+- A blog post could be written about how to use an open-source project, such as a tutorial on how to install and configure the software.
```
```suggestion
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+[How To Make a Delicious Contributing Guide]([url](https://dev.to/opensauced/how-to-make-a-delicious-contributing-guide-4bp3)https://dev.to/opensauced/how-to-make-a-delicious-contributing-guide-4bp3)
+
+- A blog post could be written about the impact of an open-source project, such as how it has been used to make a difference in the world.
```
```suggestion
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+[Retaining the best engineers]([url](https://opensauced.pizza/blog/retaining-the-best-engineers)https://opensauced.pizza/blog/retaining-the-best-engineers)
+
+- A blog post could be written on how to navigate your way around open-source as a new contributor, such as how to get started with open-source.
```
```suggestion
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+## B. Coding Type Of Contributions
+
+Coding contributions in the context of open-source software development involve writing, modifying, or improving the actual source code of an open-source project. These contributions are at the core of open-source development and are crucial for building and maintaining software
```
```suggestion
Coding contributions in the context of open-source software development involve writing, modifying, or improving the actual source code of an open-source project.
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Show reactors who reacted to highlights]([url](https://github.com/open-sauced/app/pull/1591)https://github.com/open-sauced/app/pull/1591)
```
```suggestion
[Show reactors who reacted to highlights](https://github.com/open-sauced/app/pull/1591)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Lint check happening two times on commit
+]([url](https://github.com/open-sauced/app/pull/1635)https://github.com/open-sauced/app/pull/1635)
```
```suggestion
[Lint check happening two times on commit
](https://github.com/open-sauced/app/pull/1635)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[CalcDistanceFromToday function
+]([url](https://github.com/open-sauced/app/pull/1633)https://github.com/open-sauced/app/pull/1633)
```
```suggestion
[CalcDistanceFromToday function
](https://github.com/open-sauced/app/pull/1633)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example*
+
+[Refresh token
+]([url](https://github.com/IridiumIdentity/iridium/pull/134)https://github.com/IridiumIdentity/iridium/pull/134)
```
```suggestion
[Refresh token
](https://github.com/IridiumIdentity/iridium/pull/134)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example* 
+
+[Added the contact page
+]([url](https://github.com/TechIsHiring/techishiring-website/pull/53)https://github.com/TechIsHiring/techishiring-website/pull/53)
```
```suggestion
[Added the contact page
](https://github.com/TechIsHiring/techishiring-website/pull/53)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example* 
+
+[Overflows in insights page on md screen size
+]([url](https://github.com/open-sauced/app/pull/1590)https://github.com/open-sauced/app/pull/1590)
```
```suggestion
[Overflows in insights page on md screen size
](https://github.com/open-sauced/app/pull/1590)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+
+*Example* 
+
+[Add force login component to design system
+]([url](https://github.com/open-sauced/app/pull/1330)https://github.com/open-sauced/app/pull/1330)
```
```suggestion
[Add force login component to design system
](https://github.com/open-sauced/app/pull/1330)
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,231 @@
+[Overflows in insights page on md screen size
+]([url](https://github.com/open-sauced/app/pull/1590)https://github.com/open-sauced/app/pull/1590)
+
+- **Testing and Quality Assurance**:Writing automated tests, including unit tests, integration tests, and end-to-end tests, is a coding contribution that ensures software reliability. Testing helps catch and prevent regressions and issues.
```
```suggestion
- **Testing and Quality Assurance**: Writing automated tests, including unit tests, integration tests, and end-to-end tests, is a coding contribution that ensures software reliability. Testing helps catch and prevent regressions and issues.
```

CBID2 reviewed (changes requested):
I have more suggestions for you @Lymah123

Lymah123 commented:
Yes, I have seen those suggestions. I was wondering what could be the problem before I submitted the PR . Thank you so much! @CBID2 ! I appreciate it!

Lymah123 commented:
@BekahHW, @CBID2 and @geoffreylgv 

You can review again.

Thank you.

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,232 @@
+
+   ## A. Non-Coding Type Of Contributions
+  
+  Non-coding contributions are valuable contributions made to open-source projects that do not involve writing or modifying code. These contributions are essential for the overall health and success of open-source communities and projects. They help improve documentation, user experience, project management, and community engagement.
```
```suggestion
  Non-coding contributions do not involve writing or modifying code. Contrary to popular belief, they are essential for the overall health and success of open-source communities and projects. These contributions help improve documentation, user experience, project management, and community engagement.
```

Lymah123 replied:
@CBID2 , I have made the correction.

CBID2 reviewed (changes requested):
Just made another suggestion @Lymah123

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+  - Blog writing
+
+
+### DOCUMENTATION
```
[proofreading] Avoid using all-caps when writing headings
```suggestion
### Documentation
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+
+*Contributing to documentation is a great way to help open-source projects and make them more accessible to everyone. If you have the skills and the interest, I encourage you to get involved!*
+
+### GRAPHIC DESIGN
```
```suggestion
### Graphic Design
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+[Enhance feed layout responsiveness 
+](https://github.com/open-sauced/app/pull/1553)
+
+### BLOG WRITING
+
```
```suggestion
### Blog Writing

```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+
+### BLOG WRITING
+
+Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. Here are some ways you can use blogging to contribute:
+
+- Reflect on your experiences as a contributor
+- Describe and review the project’s latest updates
+- Mentioning the project when raising awareness about certain topics or in a guide on how to get started in open source
+ 
```
[Proofreading] Since your next heading describes the examples, I think it's best to omit this part.
```suggestion
Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. 
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+- Mentioning the project when raising awareness about certain topics or in a guide on how to get started in open source
+ 
+
+### *Here are some specific examples of how blog writing can be used to contribute to open-source projects:*
```
[Proofreading] Turning this into a sentence instead of heading would be best
```suggestion
Here are some examples of blog writing contributions:*
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+
+### *Here are some specific examples of how blog writing can be used to contribute to open-source projects:*
+
+- A blog post could be written about the latest features of an open-source project, such as a new version of the software or a new plugin.
```
```suggestion
- You can write about the latest features of an open-source project, such as a new version of the software or plugin.
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+
+[Announcing the 0.2.0 release of Analog](https://dev.to/analogjs/announcing-the-020-release-of-analog-aa1)
+
+- A blog post could be written about how to use an open-source project, such as a tutorial on how to install and configure the software.
```
```suggestion
- Love teaching? You can write a tutorial on how to install and configure the software.
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+## B. Coding Type Of Contributions
+
+Coding contributions in the context of open-source software development involve writing, modifying, or improving the actual source code of an open-source project.
+Coding contributions are the heart of open-source development, as they directly impact the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Open-source projects typically have guidelines and processes for accepting and integrating coding contribution ...
```
```suggestion
These contributions are the heart of open-source development, as they directly impact the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Open-source projects typically have guidelines and processes for accepting and integrating coding contributions, such as code reviews and continuous integration.
```

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,233 @@
+Coding contributions in the context of open-source software development involve writing, modifying, or improving the actual source code of an open-source project.
+Coding contributions are the heart of open-source development, as they directly impact the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Open-source projects typically have guidelines and processes for accepting and integrating coding contribution ...
+
+### *Here are some specific examples of how coding can be used to contribute to open-source projects:*
+
```
```suggestion
Here are some examples of contributions you can make to a project with your coding skills:

```

CBID2 reviewed (changes requested):
Hi @Lymah123! :) I'm back with more suggestions! :) You're doing great! :) Keep it up! :)

Lymah123 commented:
Hi @CBID2! I have made the changes. Thank you.

CBID2 reviewed (changes requested):
Hi @Lymah123. There's still some stuff that need correcting

Lymah123 commented:
Hi @CBID2! I've worked on the suggestions.

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+*Example*
+  
+[  Incorrect Link Destination in Documentation
+](https://github.com/open-sauced/docs/pull/144)
```
```suggestion
[Incorrect Link Destination in Documentation](https://github.com/open-sauced/docs/pull/144)
```

Lymah123 replied:
@CBID2 , I've corrected it.

CBID2 reviewed (changes requested):
Keep it up @Lymah123

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,229 @@
+  
+  Non-coding contributions do not involve writing or modifying code. Non-coding contributions are essential for the sustainability and growth of open-source projects. They allow individuals with a variety of skills and backgrounds to participate and make meaningful contributions to the software and the community.
+
+  ### *Here are some common types of non-coding contributions:*
```
You missed this one @Lymah123

CBID2 commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,229 @@
+[Add More Information to Use This Template in the README
+](https://github.com/open-sauced/100-days-of-oss-template/pull/8)
+
+*Contributing to documentation is a great way to help open-source projects and make them more accessible to everyone. If you have the skills and the interest, I encourage you to get involved!*
```
And this one too @Lymah123

BekahHW commented on 10-Types-of-contributions.md:
Let's make this section 07 and rename the file to reflect that update. Make sure you don't capitalize the file name. So it should look like this:

`07-types-of-contributions.md`

Changing the name will require you to also make these updates:

- Change the names of 07-conclusion.md to be `08-conclusion.md`, `08-additional-resources.md` to `09-additional-resources.md` and `09-glossary.md` to `10-glossary.md`

BekahHW reviewed (changes requested):
When you add a section, you want it to show up in the sidebar. To do this, add it to `sidebar.md.`

You'll notice that the numbers of the sections will need to be updated there as well to reflect the new section.

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+# Types of contributions
+In this chapter, we will discuss the types of contributions that open-source projects involve.
+
+Mostly, there are two types of contributions. Which are;
```
You can remove "Which are;" and just have a colon at the end of the previous sentence

Mostly, there are two types of contributions:

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+- Coding type of contribution
+
+   ## A. Non-Coding Type Of Contributions
```
You can remove the A.

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+### Documentation
+
+Are you a lover of words? Do grammar errors irritate you, like seeing pineapple on pizza? Then, documentation is for you! 
```
This is great. Let's also add the following after the pineapple line:

Do you like breaking down complex problems or helping people learn?

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+- **Read the documentation**: This will help you understand how the product is used.
+  
+- **Find a task that's a good fit for your skills and interests**: If you're not sure where to start, ask the project's maintainers for suggestions.
```
I would recommend looking at the issues or discussions before reaching out to a maintainer.

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+*Example*
+
+[Create a Bootstrap Example ](https://github.com/Mirrorful/mirrorful/pull/321)
```
This looks like an example and not a readme. You can use Open-Sauced App as an example of a Readme: https://github.com/open-sauced/app

BekahHW replied:
It seems like the examples aren't linking up to the bullet points that I expect. 

I wonder, too, if it makes more sense to link the finished product ( readme, blogpost, chart) rather than the issue. Or maybe both.

Lymah123 replied:
It seems like the examples aren't linking up to the bullet points that I expect.

I wonder, too, if it makes more sense to link the finished product ( readme, blogpost, chart) rather than the issue. Or maybe both.

@BekahHW and @CBID2 , what examples would you suggest I should use here?

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+- **Be patient and persistent**: It takes time to write good documentation. Don't be afraid to ask for help if you get stuck.
+  
+
+#### Here are some specific examples of documentation contributions:
```
There's a lot of white space in this section. Let's address that by putting example and the link on the same line.


Example: [Create a Bootstrap Example ](https://github.com/Mirrorful/mirrorful/pull/321)

BekahHW replied:
I also wonder if it would be easier to follow if it were a numbered list. Or even a chart. 

@CBID2 what do you think?

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+*Example*
+
+[Insights on social card generation](https://github.com/open-sauced/opengraph/pull/51)
```
I don't think this matches up with logo and branding. It's ok if you can't find an example. That's tricky.

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+*Example* 
+
+[Enhance feed layout responsiveness 
+](https://github.com/open-sauced/app/pull/1553)
```
This is still in the non-coding section, but a lot of the examples are coding. I think the most straightforward thing to do would be to remove the examples. You can have a list of different types. This might be helpful: https://dev.to/opensauced/how-to-contribute-to-open-source-without-knowing-how-to-code-a-guide-with-project-suggestions-59e5

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+[Enhance feed layout responsiveness 
+](https://github.com/open-sauced/app/pull/1553)
+
+### Blog Writing
```
I think one important thing to mention here is that writing about an oss project creates more visibility for the project and the maintainer, which can lead to better discoverability and support.

BekahHW commented on 10-Types-of-contributions.md:
```diff
@@ -0,0 +1,228 @@
+
+### Ways to use Blogging as a Open Source Contribution
+
+- You can write about the latest features of an open-source project, such as a new version of the software or a new plugin.
```
See the comment in one of the previous sections about removing whitespace.

I would stick to examples of different types listed.

1. blog post
2. tutorial
3. Product announcement
4. how to use the product guide

Lymah123 replied:
@BekahHW , where do you suggest I should put these lists?

Thanks.

BekahHW commented:
@Lymah123 thanks for all your hard work on this. It's getting closer!

Lymah123 commented:
Alight @BekahHW 

Thank you!

Lymah123 commented:
@BekahHW and @CBID2 . I have made necessary changes on your suggestions. Waiting for your feedback.
Thank you!

CBID2 commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,174 @@
+
+### Blog Writing
+
+Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. 
```
[nitpick] I gave the beginning a little bit of humor. It can help appeal to contributors with this skill.
```suggestion
If you're more of a creative writer, then blog writing is your perfect match.This type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. 
```

CBID2 commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,174 @@
+
+### Graphic Design
+
+Graphic design is a type of open-source contribution that involves creating visual assets, such as logos, icons, illustrations, website designs, and other graphical elements, for open-source projects. These visual assets play a significant role in enhancing the overall user experience and branding of the project. Graphic design contributions are important because they can make open-source softwar ...
```
```suggestion
Love drawing or painting? Consider doing Graphic design. This open-source contribution involves creating visual assets, such as logos, icons, illustrations, website designs, and other graphical elements, for open-source projects. These visual assets play a significant role in enhancing the overall user experience and branding of the project. Graphic design contributions can make open-source software more appealing, user-friendly, and recognizable.
```

CBID2 commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,174 @@
+
+## Coding Type Of Contributions
+
+These contributions in the context of open-source software development involve writing, modifying, or improving the actual source code of an open-source project.
+Coding contributions are the heart of open-source development, as they directly impact the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Open-source projects typically have guidelines and processes for accepting and integrating coding contribution ...
```
```suggestion
Coding contributions are the heart of open-source development. They entail writing, modifying, or improving the actual source code of an open-source project,directly impacting the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Also, open-source projects typically have guidelines and processes for accepting and integrating coding contributions, such as code reviews and continuous integration.
```

CBID2 reviewed (changes requested):
Just added more changes @Lymah123

Lymah123 commented:
@CBID2 , I have made the changes.

geoffreylgv commented:
> @CBID2 , I have made the changes.

Good job @Lymah123, you're doing great.

Lymah123 commented:
Thank you @geoffreylgv .  I appreciate you.

Lymah123 commented:
@BekahHW and @CBID2 .

I would like to know the status of my PR.

Thanks.

BekahHW commented:
Hey @Lymah123 the OpenSauced team is doing a company retreat, so we'll respond by Monday.

Lymah123 commented:
Alright. 

Thank you @BekahHW .

BekahHW commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,173 @@
+
+Example: [Open Source 101: A Beginner's Guide to Getting Started](https://opensauced.pizza/blog/open-source-101-a-beginner's-guide-to-getting-started)
+
+## Coding Type Of Contributions
```
I think we should change it to Coding and Non-coding Types of Contributions

BekahHW commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,173 @@
+
+## Coding Type Of Contributions
+
+Coding contributions are the heart of open-source development. They entail writing, modifying, or improving the actual source code of an open-source project, directly impacting the functionality and quality of the software. Contributors collaborate with project maintainers and the community to ensure that their code contributions align with project goals and coding standards. Also, open-source pr ...
```
I have a hard time thinking of it as the heart. And tbh, I may have said this before, but I think we should adjust our language here.

Coding contributions power open-source development.

Lymah123 commented:
@BekahHW , I have made the changes.

BekahHW commented on 07-Types-of-contributions.md:
The file title still needs to be updated. If you look at the titles, you'll notice the first word in the title is lowercase. So in this case, it should be `07-types-of-contributions.md`. 

![screenshot of titles](https://github.com/open-sauced/intro/assets/34313413/100acdd6-b549-46d9-8d9e-d663d03dbf7f)

BekahHW reviewed (changes requested):
You still need to add this to `_layouts/sidebar.md`, and update the section numbers in there as well. If you need help with that, let me know. 

![sidebar.md](https://github.com/open-sauced/intro/assets/34313413/d4920aef-e570-47ed-8130-f4f2ff61292d)

BekahHW commented on 07-Types-of-contributions.md:
```diff
@@ -0,0 +1,173 @@
+
+- Coding type of contribution
+
+   ## Non-Coding Type Of Contributions
```
Remove the white space here.

BekahHW commented on 08-additional-resources.md:
```diff
@@ -3,13 +3,13 @@ Congratulations on your journey into open source! Here are some resources for fu
 - [Mozilla's Open Source Student Network (OSSN)](https://community.mozilla.org/en/): Provides resources and support for students interested in open source, including guides, events, and mentorship opportunities.
 
-- [Introduction to GitHub and Open-Source Projects](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) by DigitalOcean: A beginner-friendly tutorial series that covers the basics of open source, including how to find projects to contribute to, understanding the codebase, and submitting contributions.
+- [Introduction to GitHub and Open-Source Projects](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) by DigitalOcean: A beginner-friendly tutorial series that covers the basics of open source, including how to find projects to contribute to, understand the codebase, and submit contributions.
```
Good job catching this!

BekahHW reviewed (changes requested):
You've made a lot of progress. Thanks so much for all your hard work on this issue! I've made a couple more comments. If you're not sure how to do something, let me know.

One general comment is to look back at the list of examples and make sure they accurately reflect the title they fall under.

BekahHW reviewed (changes requested):
You've made a lot of progress. Thanks so much for all your hard work on this issue! I've made a couple more comments. If you're not sure how to do something, let me know.

One general comment is to look back at the list of examples and make sure they accurately reflect the title they fall under.

Oguntunjiharlimatodunola commented:
Alright. I will make the necessary corrections as suggested and look into the examples. 

Thank you!

Oguntunjiharlimatodunola commented:
Hi, @BekahHW. I have made correction to the suggestions. You can review.

CBID2 commented on 07-types-of-contributions.md:
```diff
@@ -81,7 +81,7 @@ Love drawing or painting? Consider doing Graphic design. This open-source contri
 - **Icon Design**: Icons are essential for user interfaces. Graphic designers can create custom icons for software applications or websites, improving the visual appeal and usability of the project.
 
-Example: [Redesign footer](https://github.com/open-sauced/opengraph/pull/51)
+Example: [Eventhub signup](https://github.com/WebXGuild/webx-guild/pull/13)
```
Proofreading: remember to use the `!` when linking images in Markdown. 
```suggestion
Example: ![Eventhub signup](https://github.com/WebXGuild/webx-guild/pull/13)
```

Oguntunjiharlimatodunola commented:
Alright @BekahHW .

BekahHW commented on sidebar.md:
Please delete this file and instead update `_layouts/sidebar.md`

CBID2 commented:
Hey @Lymah123. There are some merge conflicts on your branch. Can you fix this?

Lymah123 commented:
![Screenshot (1286)](https://github.com/open-sauced/intro/assets/109049037/75863bab-82f9-41a1-ba9a-38d64fd2a402)
Hi @CBID2 and @BekahHW  , I'm having issue resolving the conflicts, can you see from the screenshot? I couldn't click on the resolve conflicts.
 What would you suggest?

I have checked the blog post likewise Youtube videos nothing seems to work yet. I guess I am stuck, I need help on how to resolve the conflicts.

CBID2 commented:
> ![Screenshot (1286)](https://github.com/open-sauced/intro/assets/109049037/75863bab-82f9-41a1-ba9a-38d64fd2a402)
> 
> Hi @CBID2 and @BekahHW  , I'm having issue resolving the conflicts, can you see from the screenshot? I couldn't click on the resolve conflicts.
> 
>  What would you suggest?
> 
> 
> 
> I have checked the blog post likewise Youtube videos nothing seems to work yet. I guess I am stuck, I need help on how to resolve the conflicts. 

Have you tried this tool @Lymah123: https://blog.kaleidoscope.app/2022/04/12/resolving-merge-conflicts-in-vscode-with-kaleidoscope/

Lymah123 commented:
@CBID2 , I have been able to resolved the conflicts. Thank you!

CBID2 reviewed (approved):
LGTM

CBID2 commented:
Things look great now @BekahHW

BekahHW commented:
@Lymah123 I made some updates in this PR. Can you merge [this branch](https://github.com/open-sauced/intro/pull/59) into what you have and then I'll go ahead and merge this in.

Lymah123 commented:
@BekahHW , Done!
